### PR TITLE
Add extra-vars-key and extra-vars-value flags

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This PR is to add the option of passing any key/value pair as extra_vars to the ansible-playbook that `kubetest2-tf` internally uses to deploy k8s cluster on the VM created on PowerVS.

https://github.ibm.com/powercloud/container-dev/issues/1476

Thanks to @Karthik-K-N for the idea of using `Maps` as an intermediatory to store the values :) 

**Verification of code:**
**_Command passed:_**
kubetest2 tf --powervs-dns k8s-tests --powervs-image-name centos-83-12142020 --powervs-region osa --powervs-zone osa21 --powervs-service-id 083d5bf8-c41a-4f3c-8713-f5fdfb4d4b5b --powervs-ssh-key powercloud-bot-key --ssh-private-key "C:\Users\RajalakshmiGirish\Desktop\OPENSHIFT PROJECT\git.ibm keys\itcci-bot.txt"  --extra-vars-key build_version --extra-vars-value v1.19.2 --extra-vars-key runtime --extra-vars-value containerd --cluster-name "test-extra-vars" --up --powervs-memory "16" --auto-approve

**_Trace/Log during execution:_**
```
I0531 01:09:48.147852   20492 deployer.go:205] commonJSON: {"release_marker":"ci/latest","build_version":"","cluster_name":"test-extra-vars","apiserver_port":992,"workers_count":0,"bootstrap_token":"pec9da.ygbbv3jvz8io3ggn","kubeconfig_path":"C:\\Users\\RajalakshmiGirish\\test-extra-vars\\kubeconfig","ssh_private_key":"C:\\Users\\RajalakshmiGirish\\Desktop\\OPENSHIFT PROJECT\\git.ibm keys\\itcci-bot.txt","extra_cert":"163.68.64.202"}
I0531 01:09:48.150551   20492 deployer.go:218] finalJSON with extra vars: {"apiserver_port":992,"bootstrap_token":"pec9da.ygbbv3jvz8io3ggn","build_version":"v1.19.2","cluster_name":"test-extra-vars","extra_cert":"163.68.64.202","kubeconfig_path":"C:\\Users\\RajalakshmiGirish\\test-extra-vars\\kubeconfig","release_marker":"ci/latest","runtime":"containerd","ssh_private_key":"C:\\Users\\RajalakshmiGirish\\Desktop\\OPENSHIFT PROJECT\\git.ibm keys\\itcci-bot.txt","workers_count":0}
I0531 01:09:48.308833   20492 ansible.go:27] ansible-playbook with args: [--inventory=test-extra-vars\hosts --extra-vars={"apiserver_port":992,"bootstrap_token":"pec9da.ygbbv3jvz8io3ggn","build_version":"v1.19.2","cluster_name":"test-extra-vars","extra_cert":"163.68.64.202","kubeconfig_path":"C:\\Users\\RajalakshmiGirish\\test-extra-vars\\kubeconfig","release_marker":"ci/latest","runtime":"containerd","ssh_private_key":"C:\\Users\\RajalakshmiGirish\\Desktop\\OPENSHIFT PROJECT\\git.ibm keys\\itcci-bot.txt","workers_count":0} test-extra-vars\install-k8s.yml]
```

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/36#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: kubetest2-tf/deployer/deployer.go
Comment (mkumatag) [inline kubetest2-tf/deployer/deployer.go:115]: explore `StringToStringVarP` function which does the key-value pair mapping automatically from the command line.

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: kubetest2-tf/deployer/deployer.go
Comment (Rajalakshmi-Girish) [reply kubetest2-tf/deployer/deployer.go]: Below worked after changing to `StringToStringVar`
```
 --extra-vars "build_version=v1.19.2,runtime=containerd,bucket=kubernetes-release,directory=release,release_marker=release/v1.19.2"
```

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/36#pullrequestreview-672387618" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/kubetest2-plugins/pull/36#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Fkubetest2-plugins).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/kubetest2-plugins/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
